.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: read node.js version
+        id: nodejs-version
+        run: |
+          content=`cat ./.nvmrc`
+          echo "::set-output name=content::$content"
+      - uses: actions/setup-node@v2
+        with:
+          # See https://github.com/actions/setup-node/issues/32
+          node-version: ${{ steps.nodejs-version.outputs.content }}
+      - run: npm install
+      - run: npm test
+      - uses: JS-DevTools/npm-publish@v1
```

Comment:
Do we need to run `compile` before publishing to create the `out/` directory?

Reply:
Because the package.json has a [prepare script](https://github.com/hashicorp/js-releases/blob/main/package.json#L11), both test and compile are called during install. I think that covers it?

![image](https://user-images.githubusercontent.com/272569/148242671-a598f340-bb7b-42f3-86e4-79b85d256a12.png)

Reply:
Cool! Do we want to remove `run: npm test` in the workflow because the install is already running the tests?

Reply:
Done!